Dockerfile

```diff
@@ -1,8 +1,30 @@
-FROM vmearl/sideboard:latest
+FROM ghcr.io/magfest/sideboard:main
 MAINTAINER RAMS Project "code@magfest.org"
 LABEL version.rams-core ="0.1"
 
-# add our code
-COPY . plugins/uber/
-# go ahead and install base dependencies
+# install ghostscript and gettext-base
+RUN apt-get update && apt-get install -y ghostscript gettext-base vim && rm -rf /var/lib/apt/lists/*
+
+ADD . plugins/uber/
```

Comment:
Would it be good to ADD just the requirements files, then install the dependencies, then add the rest? And also move the ENV commands up above everything else? I'm thinking of reducing the number of layers that get rebuilt on code changes. The uber-wrapper file almost never changes either so that can go up top, I think.

Reply:
Good catch! I moved that to the end, so now all the ADDs and metadata stuff happens right at the end. At least on my machine uber code changes build from cache in ~250ms now.